pyanaconda/network.py

```diff
@@ -108,6 +109,9 @@ def is_valid_hostname(hostname):
     if len(hostname) > 255:
         return (False, _("Host name must be 255 or fewer characters in length."))
 
+    if local and hostname[-1] == ".":
+        return (False, "Local host name must not end with period '.'.")
```

Comment:
This string should be translated. Use: `_("Local host name must not end with period '.'.")`